src/coreclr/jit/morph.cpp

```diff
@@ -13430,6 +13492,11 @@ PhaseStatus Compiler::fgMorphBlocks()
         INDEBUG(fgSafeBasicBlockCreation = false;);
         INDEBUG(fgSafeFlowEdgeCreation = false;);
 
+        // We will track which blocks become unreachable during morph
+        //
+        EnsureBasicBlockEpoch();
+        BlockSet unreachable = BlockSetOps::MakeEmpty(this);
```

Comment:
This can use post-order number based bit vectors.

Reply:
Good point. 

I was actually thinking I could possibly just rely on edge removal, though there are some odd cases where we artificially bump up ref counts to keep blocks from being considered unreachable.

SPMI results are way more variable across ABI/ISA than I'd expect. Not sure if this is missing collections or something more fundamental...